nautobot/project-static/js/forms.js

```diff
@@ -127,6 +129,51 @@ function initializeSlugField(context){
     }
 }
 
+function initializePositionField(context){
+    var position_field = document.getElementById('id_position');
+    if (!position_field) {
+        var position_field = document.getElementById('id_position_pattern');
+    }
+    if (position_field.length != 0) {
+        var source_arr = position_field.getAttribute('source').split(" ");
+        var length = position_field.getAttribute('maxlength');
+        if (position_field.value) {
+            position_field.setAttribute('_changed', true);
+        } else {
+            position_field.setAttribute('_changed', false);
+        }
+        position_field.addEventListener('change', function() {
+            if (!position_field.value) {
+                position_field.setAttribute('_changed', false);
+            } else {
+                position_field.setAttribute('_changed', true);
+            }
+        });
+        function repopulate() {
+            let str = "";
+            for (source_str of source_arr) {
+                if (str != "") {
+                    str += " ";
+                }
+                let source_id = 'id_' + source_str;
+                let source = document.getElementById(source_id)
+                str += source.value;
+            }
+            position_field.value = str.slice(0, length ? length : 100);
+        };
+        for (source_str of source_arr) {
+            let source_id = 'id_' + source_str;
+            let source = document.getElementById(source_id);
+            source.addEventListener('keyup', function() {
+                if (position_field && position_field.getAttribute('_changed')=="false") {
+                    repopulate();
+                }
+            });
+        }
+        this_context.find('button.reslugify').click(repopulate);
```

Comment:
Should that be `context` rather than `this_context`? Note that this line is still using jquery either way :)